Aquarius/Src/Aquarius/Core/Layer.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include "Aquarius/Core/Utility.h"
+
+#include <deque>
+#include <string>
+
```

Comment:
Add newline here so that there are two empty lines between last include and namespace.